test/models/partner_test.rb

```diff
@@ -6,6 +6,7 @@ class PartnerTest < ActiveSupport::TestCase
   setup do
     @new_partner = build(:partner, address: create(:address))
     @user = create(:user)
+    @new_partner.accessed_by_user = @user
   end
```

Comment:
lol. tiniest nitpick and im already sorry for it. but its probably cleaner to swap the user creation first and include it into the build arguments directly?